biome.json

```diff
@@ -40,9 +40,7 @@
         "noRestrictedImports": {
           "level": "warn",
           "options": {
-            "paths": {
-              "sentry/types": "Please import directly. For example: import type {Organization} from 'sentry/types/organization'"
-            }
+            "paths": {}
           }
         }
```

Comment:
i think its no longer used

Reply:
was just gonna leave it if we wanted to easily add more, idk